Kokkos 5 setup PR in IPPL breaks OPALX

Compiling with newest IPPL master doesn't work anymore:
```text
› [ 56%] Building CXX object src/CMakeFiles/opalx.dir/PartBunch/FieldSolver.cpp.o
› .../opalx/src/PartBunch/FieldSolver.cpp:217:17: error: ‘HostMirror’ is not a member of ‘Field_t<3>’ {aka ‘ippl::Field<double, 3, ippl::UniformCartesian<double, 3>, Cell>’}
›   217 |     Field_t<3>::HostMirror field_hostV = field->getHostMirror();
›       |                 ^~~~~~~~~~
› .../opalx/src/PartBunch/FieldSolver.cpp:218:23: error: ‘field_hostV’ was not declared in this scope
›   218 |     Kokkos::deep_copy(field_hostV, fieldV);
›       |                       ^~~~~~~~~~~
› make[2]: *** Waiting for unfinished jobs....
› make: *** [Makefile:146: all] Error 2
```

This is because of the Kokkos 5 setup that was done in IPPL-framework/IPPL#480. Should be a quick fix, will do a PR in a bit.

--- a/src/Main.cpp
+++ b/src/Main.cpp
@@ -17,6 +17,10 @@
 
 #include <cstring>
 
+// Ensure real MPI types are defined before including H5hut.
+// This avoids H5hut's serial stubs redefining MPI_Comm/MPI_Datatype.
+#include <mpi.h>
+
 extern "C" {
   #include "H5hut.h"
 }

--- a/src/PartBunch/FieldSolver.cpp
+++ b/src/PartBunch/FieldSolver.cpp
@@ -213,8 +213,8 @@ void FieldSolver<double,3>::dumpScalField(std::string what) {
     auto spacing  = mesh_mp->getMeshSpacing();
     auto origin   = mesh_mp->getOrigin();
 
-    Field_t<3>::view_type fieldV       = field->getView();
-    Field_t<3>::HostMirror field_hostV = field->getHostMirror();
+    Field_t<3>::view_type fieldV             = field->getView();
+    Field_t<3>::host_mirror_type field_hostV = field->getHostMirror();
     Kokkos::deep_copy(field_hostV, fieldV);     
 
     std::filesystem::path file(dirname);

